apis/kueue/v1beta1/clusterqueue_types.go

```diff
@@ -346,6 +350,14 @@ type ResourceUsage struct {
 	Borrowed resource.Quantity `json:"borrowed,omitempty"`
 }
 
+type FairSharingStatus struct {
+	// WeightShare represent the maximum of the ratios of usage above nominal
+	// quota to the lendable resources in the cohort, among all the resources
+	// provided by the ClusterQueue.
```

Comment:
And divided by the weight

Reply:
Also, for a weight of zero, this will return MAXINT64 (put the exact number in the comment).

Reply:
Added